config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+
+  get 'books', to: 'book#index'
```

Comment:
Please use the resource to configure your routes.
[Here some help](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default)